client/app/bundles/course/assessment/assessments-edit.jsx

```diff
@@ -4,27 +4,27 @@ import ProviderWrapper from 'lib/components/ProviderWrapper';
 import storeCreator from './store';
 import EditPage from './pages/EditPage';
 
-const mountNode = document.getElementById('assessment-edit');
-if (mountNode) {
-  const dataAttr = mountNode.getAttribute('data');
-  const data = JSON.parse(dataAttr);
-  const store = storeCreator({});
+$(document).ready(() => {
+  const mountNode = document.getElementById('assessment-edit');
```

Comment:
Is there a reason why you are shifting all the code under `$(document).ready(fn)`?